cyberduck/support.md

```diff
@@ -1,77 +1,57 @@
 Support
 ===
 
-# Application Support Folder
-
-Inside the application support folder, the application saves files needed for their operations e.g. settings, log data, history files, etc.
-
-`````{tabs}
-````{group-tab} macOS
+# Get Support
 
-You can reach the application support folder by choosing `Go → Go to folder` within the *Finder* menu, copying the path below into the appearing window, and clicking on the *Open* button afterward.
+## Feature Requests
```

Comment:
Shouldn't the support part be listed before the features part? In my opinion, the support part should have a higher priority.

Reply:
No, because a public conversation about a feature or issue is preferred to a private, unpaid support request.